app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,8 +2,13 @@
   <div id="comments" data-updated-at="<%= Time.current %>">
     <% if @article.show_comments %>
       <div class="comments-container-container">
+        <%# TODO: Fix margins, vertical centering %>
+        <div class="flex justify-between ml-7 mr-8">
+          <span class="fs-xl">Discussion</span>
```

Comment:
```suggestion
          <span class="fs-2xl fw-bold">Discussion</span>
```

Bumping up the font size and making it a little bit bolder.

Reply:
Done